test-conf/samples/hello_world/sample.yaml

```diff
@@ -14,6 +14,10 @@ common:
     regex:
       - "Hello World! (.*)"
 tests:
+  sample.basic.helloworld:
```

Comment:
Can we run hello world without RTT yet? I think we need the `BH_ARC` lib on to get PCIe to come up

Reply:
good catch